pkg/app/piped/cloudprovider/terraform/terraform.go

```diff
@@ -103,19 +101,30 @@ func (r PlanResult) NoChanges() bool {
 	return r.Adds == 0 && r.Changes == 0 && r.Destroys == 0
 }
 
+func GetExitCode(err error) int {
+	if err == nil {
+		return 0
```

Comment:
I wonder which value is better to set here. `0` or `2`.
Because we are using the `-detailed-exitcode` flag, so with the new version of terraform, we expect `err` will not be nill, right?
So IMO, when err is nil, returning `2` is safer than `0`.
How do you think about this?

Reply:
even with `-detailed-exitcode` the `err` will be `nil` if the plan commands succeeds and produces plan with no changes. 

`-detailed-exitcode` only gives us ability to differentiate between actual plan error (status code 1) and plan witch changes (status code 2). 

So I would keep this as is